x-pack/plugins/security_solution/public/common/containers/matrix_histogram/index.ts

```diff
@@ -137,12 +137,12 @@ export const useMatrixHistogram = ({
                   buckets: histogramBuckets,
                 }));
                 endTracking('success');
-                searchSubscription$.current.unsubscribe();
+                searchSubscription$.current?.unsubscribe();
```

Comment:
TS indicated that `searchSubscription$.current` cannot be undefined, so adding an extra `?` does not seem appropriate. Can you fix the test instead of the application code, please?